src/cmd/manage_nsfs.js

```diff
@@ -581,20 +569,21 @@ async function verify_delete_account(account_name) {
     });
 }
 
-async function get_account_status(data, show_secrets) {
+async function get_account_status(data, show_secrets, argv) {
```

Comment:
@shirady any reason why we need to add `argv` also to the func parameter, can we get the name from `data`? And will it give the wrong value when the user set `--name` without any value and `--access_key` with value in the update command?

Reply:
@naveenpaul1 I removed my changes and back to the original (I still think those functions should be refactored).